tests/atoms/gaia_apps.js

```diff
@@ -279,10 +279,12 @@ var GaiaApps = {
       };
 
       let displayedApp = GaiaApps.getDisplayedApp();
-      if (displayedApp.manifestURL == manifestURL ||
+      if ((displayedApp.manifestURL == manifestURL &&
+          displayedApp.entryPoint == entryPoint) ||
           (!displayedApp.manifestURL && displayedApp.origin == origin)) {
         console.log('app with origin \'${displayedApp.origin}\' and ' +
-          'manifestURL \'${displayedApp.manifestURL}\' is already running');
+          'manifestURL \'${displayedApp.manifestURL}\' and ' +
```

Comment:
Thanks, I should have checked this code. Also need to check this with jslint, because that is being checked by treeherder as well.